.storybook/preview.js

```diff
@@ -18,10 +20,56 @@ export const lightTheme = {
   globals
 }
 
-const themes = [
-  darkTheme,
-  getToggledTheme(lightTheme)
-]
+const getTheme = (theme) => {
+  if (theme == 'dark') {
+    return darkTheme
+  }
+  if (theme == 'light') {
+    return getToggledTheme(lightTheme)
```

Comment:
is there a deeper reason why only the light theme has the toggletheme method?

Reply:
the getToggledTheme method toggles the theme colors from black to white and so on, so the one theme other than the default theme which should be the opposite concerning white, black and grey-tones needs to be toggled